schema/deploy/computed_columns/form_change_payment_percentage.sql

```diff
@@ -9,16 +9,28 @@ create or replace function cif.form_change_payment_percentage(fc cif.form_change
 returns numeric as
 $fn$
 
-  select
```

Comment:
Can't comment on unchanged lines, but comment on line 4 should be updated with the 0,100 constraint (or deleted, I don't think we usually put the formula in comments)